app/assets/stylesheets/navigation.scss

```diff
@@ -6,53 +6,51 @@ nav {
 
   background-color: $bg-color;
 
-  .button-collapse {
-    i {
-      color: $font-color;
-      font-size: 1.8rem;
-      margin-right: 1rem;
-    }
+  .container {
+    width: 100%;
+    margin: 0 auto 0 15px;
   }
 
-  .brand-logo {
+  a.brand-logo {
+    padding: 0 15px;
     color: $font-color;
     font-size: 1.2rem;
     font-weight: 700;
   }
-
-  ul.right {
-    i { color: lighten($secondary-color, 25%); }
-  }
 }
 
-li.avatar {
-  img {
-    vertical-align: middle;
-    width: $nav-avatar-size;
-    height: $nav-avatar-size;
-    border-radius: 0.3rem;
-  }
-
-  i {
-    margin-left: 3px;
-  }
-}
-
-.side-nav {
-  $bg-color: color("grey", "lighten-3");
-  $font-color: $primary-font-color;
+.side-nav.fixed {
+  @extend .z-depth-0;
+  z-index: 1;
+  top: 65px;
+  width: $side-nav-width;
 
+  $bg-color: lighten($secondary-color, 15%);
+  $font-color: #fff;
   $line-height: 64px;
 
   background-color: $bg-color;
 
-  a {
+  a { color: $font-color; }
+
+  i {
     color: $font-color;
+    line-height: $line-height;
   }
 
-  i { line-height: $line-height; }
+  li:hover { background-color: lighten($bg-color, 20%); }
+
+  li.divider { background-color: darken($bg-color, 5%); }
+
+  li.avatar img {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3